docs/project/development-guidelines.md

```diff
@@ -0,0 +1,247 @@
+---
+id: development
+title: Development Best Practices
+sidebar_label: Development Best Practices
+slug: /project/development-guides
+custom_edit_url: null
+---
+
+## Defining the name of your usecase
+
+The name of the usecase must be something that refers to the business, making its real purpose clear to the business team.
+
+
+
+**Good descriptions:**
+
+✅ Search one or several funds
+
+✅ Filter users by document
+
+✅ Update unit price
+
+✅ Find the funds in Bank
+
+
+
+**Bad descriptions:**
+
+❌ list the information
+
+❌ Calculates the percentage of the fund
+
+❌ calculates the total
+
+❌ Save user in bank
+
+❌ Update Client
+
+❌ Starts the services and returns the JobId
+
+
+Reference: [https://herbsjs.org/docs/usecase/features/#creating-a-use-case](https://herbsjs.org/docs/usecase/features/#creating-a-use-case)
+
+## **Creating custom methods in the repository**
+
+**Working with PostgreSQL**
+To create a non-standard / custom query for your repository, we can call knex directly, without the need to create a new variable and call it in the constructor.
+
+**Don't:**
+
+By creating the variable this.conn and receiving the connection, we have already passed it to the mother class's knex.
```

Comment:
It is not clear in the description why this is not correct.
Also it is not clear what is this "mother class" concept.